sdk/core/azure-core-experimental/azure/core/experimental/transport/_httpx.py

```diff
@@ -124,6 +155,14 @@ def __exit__(self, *args) -> None:
         self.close()
 
     def send(self, request: HttpRequest, **kwargs) -> HttpXTransportResponse:
+        """Send a request and get back a response.
+
+        :param request: The request object to be sent.
+        :type request: ~azure.core.rest.HttpRequest
```

Comment:
Do we support transport HttpRequest?

Reply:
yes it does support transport HttpRequest as well.

Reply:
update typing to support `LegacyHttpRequest`